app/src/componentsV2/ContentList/components/ContentListTable/ContentListTable.tsx

```diff
@@ -47,6 +47,8 @@ const ContentListTable = <DataType extends { [key: string]: any }>({
         updatedExpandedRowKeys.delete(record[rowKey]);
       }
       setExpandedRowsKeys(Array.from(updatedExpandedRowKeys));
+
+      // FIXME: localStorage gets overrided by the another instance of the table
```

Comment:
Please fix this in a separate PR first

Reply:
Did we fix this?

Reply:
https://github.com/requestly/requestly/pull/1667